src/test/components/ProfileCallTreeView.test.js

```diff
@@ -178,28 +179,32 @@ describe('calltree/ProfileCallTreeView navigation keys', () => {
 
     const { profile } = getProfileFromTextSamples(profileString);
     const store = storeWithProfile(profile);
-    const callTree = mount(
+    const { container } = render(
       <Provider store={store}>
         <ProfileCallTreeView hideThreadActivityGraph={true} />
       </Provider>
     );
 
     // Assert that we used a large enough bounding box to include all children.
-    const renderedRows = callTree.find(
+    const renderedRows = container.querySelectorAll(
       '.treeViewRow.treeViewRowScrolledColumns'
     );
-    const expectedRows = callTree.find('VirtualList').prop('items');
-    expect(renderedRows.length).toBe(expectedRows.length);
+    expect(renderedRows.length).toBe(expectedRowsLength);
```

Comment:
Instead of programmatic finding the expected number which was difficult with this library, I decided to pass it as a parameter to the setup function instead because we should know this number in the test. Less magic :)
Tell me what you think.

Reply:
wfm